docs/plans/2026-03-05-gitzy-squash-command.md

```
@@ -0,0 +1,218 @@
+| `--co-author <coAuthor...>` | `-c`  | add co-authors (repeatable)                                    |
+| `--dry-run`                 | `-D`  | show what would happen without touching git                    |
+| `--no-verify`               | `-V`  | skip git hooks                                                 |
+| `--json`                    | `-j`  | output structured JSON                                         |
+| `--no-emoji`                |       | disable emoji in commit message                                |
+| `--stdin`                   |       | read answers from stdin as JSON                                |
+
+---
+
+## Command Flow
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)